sentry/content.md

```diff
@@ -8,19 +8,41 @@ Sentry is a realtime event logging and aggregation platform. It specializes in m
 
 # How to use this image
 
-## start a sentry instance
+## how to setup a full sentry instance
 
-### PostgreSQL database (as recommended by upstream)
+1.	start a redis container
 
-	docker run --name some-sentry --link some-postgres:postgres -d sentry
+		docker run -d --name some-redis redis
 
-### MySQL database
+2.	start a MySQL or PostgreSQL(recommended by upstream) database container (later steps assume PostgreSQL, replace the `--link` entries appropriately for MySQL)
 
-	docker run --name some-sentry --link some-mysql:mysql -d sentry
+		# postgres:
+		docker run -d -e POSTGRES_PASSWORD=secret -e POSTGRES_USER=sentry --name some-postgres postgres
+		# or mysql:
+		docker run -d -e MYSQL_ROOT_PASSWORD=secret -e MYSQL_DATABASE=sentry --name some-mysql mysql
```

Comment:
Probably change that note to explicitly say something more like "replace the later `--link some-postgres:postgres` with `--link some-mysql:mysql`".